src/store/modules/gateway/actions.ts

```diff
@@ -0,0 +1,59 @@
+import { Commit, ActionPayload } from 'vuex';
+import * as mutation from './types-mutations';
+import * as consts from '../../../consts/general';
+import {
+  GatewayState,
+  RootState,
+  AppGatewayServiceRequest,
+  AppGatewayServiceClient,
+  grpc,
+} from '../../types';
+
+
+interface ActionContext {
+  state: GatewayState;
+  rootState: RootState;
+  commit: Commit;
+  payload?: ActionPayload;
+}
+
+export const setNewClient = ({ state, commit }: ActionContext): any => {
+  const hostname: string = process.env.VUE_APP_HOST_NAME == null ? '' : process.env.VUE_APP_HOST_NAME;
```

Comment:
Another JS feature, you prolly interested with. I really do not like reading that logical operator `== null` also comparing with `undefined`. I really have a bad feeling about it.

`const hostname: string = process.env.VUE_APP_HOST_NAME || '';`

Reply:
I think `||` and `!!` exist for reasons like this.

Reply:
Normally, there is another reviewer to settle differences. Maybe we can get opinion from @yeseulan? She worked at Disney as a frontend dev using Angular.

Reply:
Alright. You guys have industry level experience I don't.